{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/static_source/sass/base/_typography.scss

```diff
@@ -43,7 +43,8 @@ p {
 }
 
 a {
-  @include transition(color 0.1s linear);
+  -webkit-transition: color 0.1s linear;
```

Comment:
I don't think we need to prefix these - we're going to use autoprefixer to do it